### DISCUSSION TITLE
Increase Reset button size

### DISCUSSION THREAD

**Discussion by Giganzo** [godotengine/godot-proposals#10265] [Editor]
The reset button feels a bit on the small side, which leads to some missed clicks.
As an example, this would not hit the button:

![Screenshot_20240722_171620](https://github.com/user-attachments/assets/465305f9-b50c-41ec-bdb5-0a52ada5e60e)
![Screenshot_20240722_171420](https://github.com/user-attachments/assets/2df21bd0-eb1e-4c90-9681-0e1cc6a350bd)

Do you think the size needs to be increased?

**Comment by Calinou** [comment]
That's a good idea. It should be feasible to increase its clickable area without increasing its physical size (so the icon doesn't look too large).

**Reply by Giganzo** [reply]
I added some padding to the revert ("reset") button 

[Screencast_20240725_115514.webm](https://github.com/user-attachments/assets/cf293acc-b42e-496a-93f1-6a19504eda7f)

> I can add videos correctly now ;)

Should I open this as an proposal issue, or maybe pull request?

Also found another issues with the revert button. So it might not just be the size that made it miss clicks. If a tooltip is showing it eats the input, so you have to click twice to reset. Happens on both Windows and Linux. Maybe related to https://github.com/godotengine/godot/issues/73926 should I comment there or open a new bug report?

**Reply by Calinou** [reply]
> Should I open this as an proposal issue, or maybe pull request?

You can open a PR directly.

> If a tooltip is showing it eats the input, so you have to click twice to reset.

Tooltips work like this on purpose (you can click on links or select within them, or at least you should be able to). I don't know a good solution to this usability issue, other than offsetting the tooltip which we can't really do either (otherwise it'd spawn outside the mouse position).

**Reply by Giganzo** [reply]
> You can open a PR directly.

Will do that.

> Tooltips work like this on purpose (you can click on links or select within them, or at least you should be able to). I don't know a good solution to this usability issue, other than offsetting the tooltip which we can't really do either (otherwise it'd spawn outside the mouse position).

Not sure if I did understand this correctly. The tooltip dosent cover the revert button, so it is already offset.

[Screencast_20240725_155139.webm](https://github.com/user-attachments/assets/e937a363-b75b-4fe0-b035-053c15398808)

First click hides tooltip.

**Reply by Calinou** [reply]
- Pull request opened: https://github.com/godotengine/godot/pull/94742
